subprojects/packagefiles/glfw/examples/meson.build

```
@@ -0,0 +1,37 @@
+###  GLFW examples
+#@ Build check
+if opt_examples == false
```

This is not defined anywhere.

Also, the option is already a boolean, so no need to compare to `== false`
```suggestion
if not get_option('build-examples')
```